### PR TITLE
[Keymap] Smarter KC_MAKE code and removed duplicate MOD_MASK entries

### PR DESCRIPTION
**Pull request by stanrc85**
## Description

- Better reset_keyboard() code in KC_MAKE macro
- Removed duplicate MOD_MASK entries

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by vomindoraan** [comment]
I've been working on a large PR removing all duplicate definitions of mod masks. Thanks for taking the initiative and making my job a bit easier 😀

File: users/stanrc85/stanrc85.c [hunk 1]
```diff
@@ -14,23 +14,21 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
...
           #elif defined(BOOTLOADER_CATERINA)
             send_string_with_delay_P(PSTR(":avrdude"), 10);
-          #else
-            reset_keyboard();
           #endif // bootloader options
+          send_string_with_delay_P(PSTR(SS_TAP(X_ENTER)), 10);
+          reset_keyboard();
         }
-        if (temp_mod & MODS_CTRL_MASK || temp_osm & MODS_CTRL_MASK) { send_string_with_delay_P(PSTR(" -j8 --output-sync"), 10); }
+        if (temp_mod & MOD_MASK_CTRL || temp_osm & MOD_MASK_CTRL) { send_string_with_delay_P(PSTR(" -j8 --output-sync"), 10); }
```
Comment (vomindoraan) [inline users/stanrc85/stanrc85.c]: Same reasoning as above.
```suggestion
        if (mods & MOD_MASK_CTRL) {
          send_string_with_delay_P(PSTR(" -j8 --output-sync"), 10);
        }
```
I also suggest adding some line breaks to make it a bit easier on the eyes 🙂

Comment (stanrc85) [reply users/stanrc85/stanrc85.c]: Done!

File: users/stanrc85/stanrc85.c [hunk 2]
```diff
@@ -14,23 +14,21 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
       clear_mods();
       clear_oneshot_mods();
       send_string_with_delay_P(PSTR("make " QMK_KEYBOARD ":" QMK_KEYMAP), 10);
-        if (temp_mod & MODS_SHIFT_MASK || temp_osm & MODS_SHIFT_MASK ) {
+        if (temp_mod & MOD_MASK_SHIFT || temp_osm & MOD_MASK_SHIFT ) {
```
[the remark this answers is not in the document]
Comment (stanrc85) [reply users/stanrc85/stanrc85.c]: Done!  I blame Drashna, I stole it from him :)

[the remark this answers is not in the document]
Comment (drashna) [reply users/stanrc85/stanrc85.c]: lol. Yup. 

And I was the one that mentioned/recommended changing the mod mask stuff.   And I've done so with my code, already.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**